packages/storybook/cxl-ui/cxl-jobs-components/vaadin-tabs[theme=cxl-jobs-horizontal-slider].stories.js

```diff
@@ -0,0 +1,53 @@
+import '@conversionxl/cxl-lumo-styles';
```

Comment:
To make it simpler, let's delete this and just do a single story of actual usage, such as in testimonial card context in #49